installer/steps/a_setup_virtualenv.py

```diff
@@ -22,12 +22,11 @@
     venv_installed = shell("virtualenv --version")
 
     if not venv_installed.success():
-        printlog("Please install virtualenv!")
-        printlog("https://github.com/pypa/virtualenv")
-        printlog(unix_windows.VIRTUALENV_INSTALL_MSG)
-        printlog("")
-
-    fail(">>> Apparently virtualenv on Python3 ({}) does not work. Exiting!".format(unix_windows.VIRTUALENV_CMD))
+        printlog("virtualenv not installed, installing...")
+        if install("virtualenv"):
+            printlog("Installation Complete!")
+        else:
```

Comment:
Since install can't return anything else except True, the else-block cannot be executed

Reply:
okay, will rewrite it.